Form/ChoiceList/ModelChoiceList.php

```diff
@@ -20,7 +20,18 @@
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
 use Symfony\Component\PropertyAccess\PropertyPath;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ModelChoiceList class is deprecated since SF 2.7'.
```

Comment:
Indeed, and this one does not seem to be added to compilation: https://github.com/sonata-project/SonataAdminBundle/blob/7cfd7470ee2f532844e3376e6bbed5d1ef614979/DependencyInjection/SonataAdminExtension.php#L285

Should be OK I guess.